backend/src/main/java/de/neuefische/teddyscoronadiaries/googleoauth/service/GoogleOAuthService.java

```diff
@@ -0,0 +1,34 @@
+package de.neuefische.teddyscoronadiaries.googleoauth.service;
+
+import de.neuefische.teddyscoronadiaries.googleoauth.config.GoogleOAuthConfig;
+import de.neuefische.teddyscoronadiaries.googleoauth.model.VerifyTokenResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Optional;
+
+@Service
+public class GoogleOAuthService {
+
+    private final RestTemplate restTemplate;
+    private final String baseUrl = "https://oauth2.googleapis.com/tokeninfo";
```

Comment:
could make it static final 
```suggestion
    private static final String GOOGLE_TOKEN_BASE_URL = "https://oauth2.googleapis.com/tokeninfo";
```